dfsclient/src/main/java/org/hpccsystems/dfs/client/StreamOperationMessages.java

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+ *     HPCC SYSTEMS software Copyright (C) 2019 HPCC Systems®.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *******************************************************************************/
+
+package org.hpccsystems.dfs.client;
+
+import java.util.HashMap;
+import java.util.Set;
+
+public class StreamOperationMessages
+{
+    public static int CODE_UNKNOWN = -1;
+    public enum MessageType
+    {
+        Error,
+        Warning,
+        Info
+    };
+
+    private HashMap<String, Integer> map = new HashMap<>();
+    private int totalMessageCount = 0;
+
+    private static String createKey(String message, MessageType type, int code, String column, String value)

```

Comment:
Should we use an int hash code from String.hashCode() here as the key instead of the String?